AuthService/config.json.sample

```
@@ -0,0 +1,10 @@
+{
```

Probably rename to **config.sample.json** instead of **config.json.sample** so the extension is respected.

---

good point, I've always done .sample and ignore the error in vscode.  sample.json seems fitting enough.